boards/amd/acp_6_0_adsp/acp_6_0_adsp_defconfig

```diff
@@ -0,0 +1,12 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_GEN_ISR_TABLES=y
+CONFIG_GEN_IRQ_VECTOR_TABLE=y
+CONFIG_XTENSA_RESET_VECTOR=y
+CONFIG_OUTPUT_SYMBOLS=y
+CONFIG_MULTI_LEVEL_INTERRUPTS=n
+CONFIG_2ND_LEVEL_INTERRUPTS=n
+CONFIG_BUILD_OUTPUT_BIN=n
+CONFIG_CLEANUP_INTERMEDIATE_FILES=y
```

Comment:
This one should come out.  The intermediate files are actually useful for a lot of things.  This setting exists for e.g. CI applications that need to have hundreds or thousands of build trees in cloud filesystems with limited storage.  Generic builds should use the defaults, it's not something the board layer is supposed to care about.